src/View/SSViewer.php

```diff
@@ -851,13 +851,6 @@ public function setTemplateFile($type, $file)
      */
     public static function get_base_tag($contentGeneratedSoFar)
     {
-        $base = Director::absoluteBaseURL();
-
-        // Is the document XHTML?
-        if (preg_match('/<!DOCTYPE[^>]+xhtml/i', $contentGeneratedSoFar ?? '')) {
-            return "<base href=\"$base\" />";
-        } else {
-            return "<base href=\"$base\"><!--[if lte IE 6]></base><![endif]-->";
-        }
+        return '<base href="' . Director::absoluteBaseURL() . '" />';
```

Comment:
```suggestion
        return '<base href="' . Director::absoluteBaseURL() . '">';
```
Modern browsers don't require this to be self-closing. That's true of any browser that supports HTML4+